plugins/Actions/Columns/PageViewPosition.php

```diff
@@ -18,7 +18,6 @@
 class PageViewPosition extends ActionDimension
 {
     protected $columnName = 'pageview_position';
-    protected $columnType = 'MEDIUMINT UNSIGNED DEFAULT NULL';
```

Comment:
I had to move this into the schema. Otherwise it would generate an update like this:

```sql
ALTER TABLE `matomo_log_link_visit_action` ADD COLUMN `pageview_position ` MEDIUMINT UNSIGNED DEFAULT NULL;
```

That's because Matomo thinks it is a new dimension and doesn't realise the dimension was actually renamed. This `add column` alone wouldn't be a problem since the column would already exist and the failure would be ignored. However, it becomes a problem when other dimensions are also changed and the generated SQL looks like this:

```sql
ALTER TABLE `matomo_log_link_visit_action` ADD COLUMN `pageview_position ` MEDIUMINT UNSIGNED DEFAULT NULL, ADD COLUMN `other_dimension ` MEDIUMINT UNSIGNED DEFAULT NULL;;
```

If then the column `pageview_position` already exists, the SQL would fail and it would not add the second column anymore.